File: android/app/src/main/java/com/bsbarron/midschoolapp/data/model/SchoolInfo.kt
package com.bsbarron.midschoolapp.data.model

data class SchoolInfo(
    val officeCode: String,
    val officeName: String = "",
    val schoolCode: String,
    val schoolName: String,
    val schoolKind: String,
    val location: String = "",
    val jurisdiction: String = "",
    val foundation: String = "",
    val roadAddress: String = "",
    val telephone: String = "",
    val homepage: String = ""
)

File: ios/SchoolHelperIOS/Core/Models/SchoolInfo.swift
import Foundation

struct SchoolInfo: Codable, Equatable, Identifiable {
    var id: String { "\(officeCode)-\(schoolCode)" }

    var officeCode: String
    var officeName: String
    var schoolCode: String
    var schoolName: String
    var schoolKind: String
    var roadAddress: String
}
